aws-lambda-scorer-python/lambda-template/py_scorer.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import boto3
+import botocore
+import daimojo.model
+import datatable as dt
+
+s3 = boto3.resource('s3')
```

Comment:
Let's not have a global. Globals are evil, especially is their creation is not side effect free and can fail. Also we surely want a test (which is hard to do cleanly with globals). I would prefer wrapping the stuff here into an object that get's s3 upon construction. I guess we will need a global on the top-most level to represent the scorer object, but that is acceptable (as there is no way around it). Globals on the module level are imho not.

That way, the test would be relatively simple with a mock. Also, as an object, we can more easily download the mojo only once and reuse it.

Reply:
Not sure if Lambda serves traffic from one machine, in my experience most cdn don't, we might still need to re-download the file on every request or at least check that we have it on disk.

Reply:
Multiple requests run in quick succession are likely to hit an existing instance. AWS will spawn multiple when traffic is high and shut them down if it is low. Every now and then, we will have to reinitialize, but we should not do that on every request.

Please don't implement this by checking the disk file. We really want an entity that is initialized once and then reused rather than external contract about disk file being present. Note that this is a public repo and we want to keep it clean and tidy :+1:.